### PR TITLE
remove browser plugin

### PR DESCRIPTION
**Pull request by winsvega**
connects to https://github.com/ethereum/webthree-umbrella/issues/409

I just started deleting browser stuff. the build is fine at this point need a review if I am working towards the right direction

### CODE REVIEW

File: alethzero/QNatspec.cpp [hunk 1]
```diff
@@ -20,7 +20,7 @@
  */
 
 #include "QNatspec.h"
-
+/*
```
Comment (bobsummerwill) [inline alethzero/QNatspec.cpp]: Please can you change all these commenting-out blocks into deletions?   Thanks!

**Comment by chriseth** [comment]
If we remove the browser plugin, I think we can also remove natspec completely.

**Comment by bobsummerwill** [comment]
Yes, if we can, let's remove natspec entirely.    And the natspec module in libethereum too, right, @chriseth?

And change the commented-out blocks to removed code, as per my comment above, please :-)

**Comment by chriseth** [comment]
This change might bring us xenial builds without additional work.

**Pull request merged**